timescaledb/how-to-guides/continuous-aggregates/create-a-continuous-aggregate.md

```diff
@@ -161,9 +161,16 @@ GROUP BY name, bucket;
 ```
 
 <Highlight type="note">
-This also works in Timescale&nbsp;2.9 and later running PostgreSQL&nbsp;12, but
-you cannot also have a `USING` clause in your `JOIN`. If you need a `USING`
-clause, you must have PostgreSQL&nbsp;13 or later.
+The join can be between only two tables where one has to be hypertable
+and other one should be  a normal PostgreSQL table. However, the order of tables
+does not matter.
+The FROM/JOIN clause cannot have any other continous aggregate.
+The joins type can only be INNER JOIN.
+Only equality condition are allowed as the join condition.
```

Comment:
We should at least update this with whatever restrictions we add above.

But this also worries me because we have the same things twice. I would prefer if we have everything in one place and then link to that canonical section for joins on caggs from everywhere.

Reply:
Let's merge this version as it is blocking the release announcement and follow-up on that with a new PR